docs/src/components/ComponentDoc/ComponentPropsTable/ComponentPropsRow.tsx

```diff
@@ -23,11 +24,14 @@ const ComponentPropValue: React.FunctionComponent<ComponentPropType> = props =>
     const kindParam = parameters[1] && parameters[1].name !== 'never'
     const kindIsVisible = name === 'ShorthandCollection' && kindParam
 
+    const showHash = parentInfo.parentDisplayName || _.size(getComponentGroup(componentName)) > 1
+    const propsSection = showHash ? `#${_.kebabCase(componentName)}` : ''
```

Comment:
Why we can't always add hash? Is it bad?

Reply:
We don't show it in case when its the main prop of a component (e.g. ButtonProp, but not ToolbarMenuItemProp) and there are no other related props (in that case, the prop page does not have the header to navigate to, it only contains one prop table)